.github/workflows/inviwo.yml

```diff
@@ -77,8 +82,15 @@ jobs:
       with:
         repository: 'microsoft/vcpkg'
         path: 'vcpkg' # The cmake vcpkg present expects vcpkg to be next to the inviwo source folder
-    
+
+    - name: 'Bootstrap vcpkg'
+      shell: bash
+      run: |
+        ${{ matrix.bootstrap_vcpkg }} 
+        ./vcpkg/vcpkg integrate install
```

Comment:
No we do not want to use that, we use manifests and toolchains so we can control the dependencies